doc/api/stream.md

```diff
@@ -1645,13 +1645,24 @@ parent class constructor:
 const { Writable } = require('stream');
 
 class MyWritable extends Writable {
-  constructor(options) {
-    super(options);
+  constructor({ highWaterMark, ...options }) {
+    super({
+      highWaterMark,
+      autoDestroy: false,
```

Comment:
In examples this should probably be `true` unless it really needs to be `false`...